superset-frontend/src/explore/components/EmbedCodeContent.jsx

```diff
@@ -107,46 +97,42 @@ const EmbedCodeContent = ({ formData, addDangerToast }) => {
           readOnly
           css={theme => css`
             resize: vertical;
+            margin-top: ${theme.sizeUnit * 2}px;
             padding: ${theme.sizeUnit * 2}px;
             font-size: ${theme.fontSizeSM}px;
             border-radius: 4px;
             background-color: ${theme.colorPrimaryBg};
           `}
         />
       </div>
-      <div
+      <Space
+        direction="orizzontal"
```

Comment:
I guess this is Italian 😄  ?

```suggestion
        direction="horizontal"
```

Reply:
nope, only typo error, but should be changed :)